docs/docs/tutorial/contentful/index.mdx

```diff
@@ -0,0 +1,313 @@
+---
+title: "Getting started with Gatsby and Contentful"
+---
+
+<iframe
+  width="560"
+  height="315"
+  src="https://www.youtube.com/watch?v=HcBJ9t4oWlw"
+  title="YouTube video player"
+  frameborder="0"
+  allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+  allowfullscreen
+></iframe>
+
+## Pre-flight checks
+
+Before you get started, check the following.
+
+- Have a [Contentful account](https://www.contentful.com/sign-up/)
+- Have at least node v14.0 installed
+- Have Git installed
+- Are using one of the supported remote Git providers
+
+## Create a Repository
+
+Before you get going it's a good idea to create a new repository and at the end of this tutorial you'll deploy your site to Gatsby Cloud by connecting to your chosen Git Provider. You can check the supported Git Providers here: [Gatsby Cloud Supported Git Providers](https://www.gatsbyjs.com/docs/reference/cloud/supported-git-providers/).
+
+### 1. Initialize npm
+
+Run npm ` init -y` to create a default `package.json`.
+
+```shell
+npm init -y
+```
+
+This will create a `package.json` similar to this code snippet.
+
+```json
+// package.json
+
+{
+  "name": "gatsby-contentful",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC"
+  …
+}
+```
+
+### 2. Create .gitignore
+
+Create a default, pre-populated `.gitignore` file for node.
+
+```shell
+npx gitignore node
+```
+
+This will create the default node `.gitignore` file which ignores the `node_modules` directory. You might also like to add the `public` directory as you don't need to commit this.
+
+You're now ready to install the dependencies.
+
+### 3. Install Dependencies
```

Comment:
I'll address some of the above points, but the reason we're not using `npx create-gatsby` is because it does all the things you've listed above. 

This new approach to tutorials is to do as much as possible manually so folks learn. 

There are starters and CLI docs for folks who just wanna spin up a new project without learning about what each thing does.